README.md

```diff
@@ -62,8 +64,6 @@ Constraints:
   * Major version of the Data Product is always the same as the major version of all of its components and it is the same version that is shown in both Data Product ID and component ID.
 * `InfrastructureTemplateId: [String]` the id of the microservice responsible for provisioning the component. A microservice may be capable of provisioning several components generated from different use case templates. 
 * `UseCaseTemplateId: [Option[String]]` the id of the template used in the builder to create the component. Could be empty in case the component was not created from a builder template.
-* `Allows: [Array[String]]` It is an array of user/role/group related to LDAP/AD user. This field is defining who has access in read-only to this specific output port.
```

Comment:
Previously, this information defined the users/groups/dataproducts that could have read access to the outputport. Where is this information now? Has it been deleted?

Reply:
it will be provided somewhere else, we don't want to have a new version of the data product descriptor every time the allow list is changing, it would be misleading